internal/context/command_update.go

```diff
@@ -18,11 +15,10 @@ func (c *command) newUpdateCommand() *cobra.Command {
 		RunE:              c.update,
```

Comment:
Wait we should also delete this command, right? I think we have nothing to update if you can't create a Kafka-based context in the first place.

Reply:
I thought about it, but it still has some functionality: you can set the current Kafka cluster for a context.

Should we remove that in favor of `confluent kafka cluster use`? The only difference is that you'll have to switch contexts first.

Reply:
> Should we remove that in favor of confluent kafka cluster use?

Yep, I think so.

Reply:
(as long as no functionality is lost, we'd prefer having one standardized way to do this)